src/Jobs/DataTableExportJob.php

```diff
@@ -149,6 +149,10 @@
                 /** @var array|bool|int|string|null|DateTimeInterface $value */
                 $value = $row[$property] ?? '';
 
+                if (is_callable($column->exportRender)) {
+                    $value = call_user_func($column->exportRender, $value);
```

Comment:
For consistency with the buttons package, lets pass the row and value in the callback. Here is how it was used on the buttons export function.

```php
                if (isset($column->exportRender)) {
                    $callback = $column->exportRender;
                    $results[$title] = $callback($row, $data);
                } else {
                    $results[$title] = $data;
                }
```

Reply:
Good point. I just pushed the changes